TableTopCampaignManager/DiscordBot.cs

```diff
@@ -49,5 +52,47 @@ public async Task InitializeCommands()
                 Console.WriteLine(json);
             }
         }
+        private async Task SlashCommandHandler(SocketSlashCommand command)
+        {
+            switch (command.Data.Name)
+            {
+                case "roll":
+                    await HandleRollCommand(command);
+                    break;  
+            }
+        }
+        private static int[] CalculateRoll(string userInput, SocketSlashCommand command)
+        {
+            Random random = new Random();
+            int numberOfRolls = Int32.Parse(userInput.Split('d')[0]);
+            int dieValue = Int32.Parse(userInput.Split('d')[1]);
+            int[] calculatedValues = new int[numberOfRolls];
+            if (numberOfRolls > 10)
```

Comment:
let's make "10" a variable at the top level of CalculateRoll.

`int MAX_ROLLS = 10`

Then dynamically set the text below with template literals